File: schemaorg/educationalorganization.go
package schemaorg

import "encoding/json"

// EducationalOrganization see : https://schema.org/EducationalOrganization
type EducationalOrganization struct {

typeContext

Organization

// Alumni see : /alumni
// Alumni of an organization. Inverse property: alumniOf (see: https://schema.org/alumniOf).
Alumni *Person `json:"alumni"`

}

func (v *EducationalOrganization) MarshalJSON() ([]byte, error) {
	v.C = "http://schema.org"
	v.T = "EducationalOrganization"

	return json.Marshal(v)
}

File: schemaorgphp/classes/educational_organization.php
<?php

class EducationalOrganization extends Organization implements JsonSerializable {

	public static $context = 'http://schema.org';
	public static $type = 'EducationalOrganization';
	
	/**
	 * Alumni of an organization. Inverse property: alumniOf (see: https://schema.org/alumniOf).
	 * see : https://schema.org/alumni
	 * @var \Person|\Person[]
	 */
	public var $alumni;
	
	public function jsonSerialize() {
		$out = array(
			'@context' => 'http://schema.org',
			'@type' => 'EducationalOrganization'
		);
		
		$serialized = so_json_serialize( $this->alumni );
		if ( ! empty( $serialized ) ) {
			$out['alumni'] = $serialized;
		}
		
		return $out;
	}
}
